draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -134,7 +137,7 @@ The protocol described here bases its design on the following protocol requireme
 
 * Supporting insecure HTTP
 
-* Supporting legacy HTTP versions
+* Supporting HTTP versions below 2
```

Comment:
I would prefer we just remove the whole bullet item. Its vestigal from when we MUST'd h2. It turns out supporting < h2 was a requirement for the working group.